scipy/special/_basic.pyi

```diff
@@ -0,0 +1,293 @@
+from __future__ import annotations
+
+from typing import (
+    Callable,
+    overload,
+    Literal,
+)
+
+from ._ufuncs import psi
+from numpy import sinc as sinc
```

Comment:
I notice that a miss those two functions in the stub file, but I'not sure if that is the correct way of add the stub for `psi`.

```
from ._ufuncs import psi
digamma = psi
```

Reply:
This seems fine to me.